### ISSUE TITLE
Skip Notes instead of Cards

### ISSUE THREAD

**Issue by ohare93** [kaegi/MorphMan#106]
I have only just realised I have been missing the Card 2 of my language learning note type, as I had "Skip cards with fresh vocab" enabled. I believe this worked on a Note level, not a card level, and very much need it to work so. Here is an example.

# Note 1

Hun er afhængig af stoffer.
\<Danish Audio>
She is addicted to drugs

# Note 2
Han er afhængig af stoffer.
\<Danish Audio>
He is addicted to drugs

Learning both of these would obviously be a waste, as I long ago learned "he" and "she", which are not the Focus Morphs here. However the unseen side effect of skipping this second note is that it skips all other cards on the first note! 

### Front
Hun er afhængig af stoffer.
\<Danish Audio>

### Back
She is addicted to drugs

### Front
\<Danish Audio>

### Back
Hun er afhængig af stoffer.
She is addicted to drugs

I would like to hear the native language audio, in order to improve my listening skills, while also skipping the fresh comprehension cards from yet unseen notes. **Have I misunderstood something, and this in indeed possible?** Or is this the intended use case? If so, I believe I will make a fork and try and implement this as a config option, along the lines of "do not skip cards from seen notes".

Thanks for any help :+1:

**Comment by ohare93** [comment]
Is this the intended functionality? If I made a PR to add this as an option would it be pulled?

**Comment by AtilioA** [comment]
I'm pretty sure this is intended since Card 2 doesn't have unknown morphs so it should be skipped.

**Comment by AtilioA** [comment]
I'm pretty sure this is intended; since Card 2 doesn't have unknown morphs, it should be skipped.

**Comment by ohare93** [comment]
> I'm pretty sure this is intended; since Card 2 doesn't have unknown morphs, it should be skipped.
@AtilioA 

I would argue that all Cards of a Note are usually required to change a Morph from "known" to "unknown", and that is a principle of Anki and learning using Flashcards. As my example above shows I can be shown the exact same info in multiple ways, and thus that deepens the knowing.

To give another more detailed example, let's say I am learning geography and want to know the countries of the world and their capitals. If could learn that the Capital of Australia is Canberra using a note with a single card with Australia on the front and Canberra on the back. Thus if asked "What is the capital of Australia" I could answer "Canberra". However if instead I was asked "Canberra is the capital of what country?" I may have a hard time connecting it to Australia as I have only learned the information 'one way'. Learning this the other way is as simple as having a second card in the note with "Canberra" on the front and "Australia" on the back. Do I really 'know' the capital of Australia if I do not recognise it by sight, without having 'Australia' as context. Similarly, using my example above, do I really know "Hun er afhængig af stoffer." means "She is addicted to drugs" in Danish, if I cannot recognize it when heard spoken?

Some knowledge needs this multi directional learning more than others, however I believe users should have the *option* to do so. Morphman should consider morphs based on *Notes and not Cards*, thus giving people the choice to learn the other cards while maintaining the benefits of Morphman. Most Morphman decks I have seen tend to only have one card type (probably because multiple are not supported) so they would not be affected.

I have made a shitty workaround on my personal copy, but would be happy to make a PR for this if it would be accepted. Having it as a config option would be fine too, though I'd argue it should be an opt out, not an opt in.

**Comment by AtilioA** [comment]
@ohare93 Sure, don't get me wrong, I was just pointing that it seems it was not a technical oversight.
I too have listening and reading cards for the same note and would like this feature. I'd also like to know what you've changed for it to work like that if you don't mind :)

**Comment by ohare93** [comment]
@AtilioA A quick two liner, but it seems to work for me! :+1: https://github.com/ll-in-anki/MorphMan/commit/4b5265ae617fd53bb67adb47f460823fc7f8e4d7 feel free to pull this branch, or just make these two line changes yourself :+1:

**Comment by AtilioA** [comment]
> @AtilioA A quick two liner, but it seems to work for me! 👍 [ll-in-anki@4b5265a](https://github.com/ll-in-anki/MorphMan/commit/4b5265ae617fd53bb67adb47f460823fc7f8e4d7) feel free to pull this branch, or just make these two line changes yourself 👍

Seems like it worked for me as well! This needs to be included in the next MorphMan iteration. I just hope this project isn't abandoned

**Comment by ohare93** [comment]
> Seems like it worked for me as well! This needs to be included in the next MorphMan iteration. I just hope this project isn't abandoned

Lovely, glad to hear it works :+1: if you experience anything strange, do let me know. I've made a PR for this, which will hopefully get some more traction :+1: #146

**Comment by ohare93** [comment]
@AtilioA this has been fixed in the Morphman Anki-addon for Anki 2.1 :clap: see my most recent post in #146 for context

**Comment by AtilioA** [comment]
> @AtilioA this has been fixed in the Morphman Anki-addon for Anki 2.1 :clap: see my most recent post in #146 for context

Yeah, I saw that! Great to hear. Now we gotta sort out that repo stuff
